src/Analysis/GWASV2/Utils/cohortMiddlewareApi.js

```diff
@@ -76,7 +13,12 @@ export const fetchSimpleOverlapInfo = async (
   outcome,
 ) => {
   const variablesPayload = {
-    variables: [...selectedCovariates, outcome],
+    variables: [...selectedCovariates, outcome,
```

Comment:
TODO: this is a bit fragile code in case `outcome` is null. I believe it only happens in storybooks now, but more robust logic here would be nice.

Reply:
Not introduced by this PR though...maybe can be addressed in a separate ticket.

Reply:
solved.